vnext/local-cli/cli.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ */
+
+'use strict';
+
+var cli = require('@react-native-community/cli');
```

Comment:
I'm scratching my head as to why we need to do this.  Can you debug the broken scenario?  Feel free to ping me if you want help with this.

Reply:
I think I'll check this in for now as a workaround. Will remove it later when I figure out the right fix.